client/vite.config.js

```diff
@@ -0,0 +1,34 @@
+import { defineConfig } from 'vite'
+import solidPlugin from 'vite-plugin-solid'
+// import devtools from 'solid-devtools/vite';
+
+export default defineConfig({
+  plugins: [
+    solidPlugin()
+  ],
+  server: {
+    port: 3000,
+    proxy: {
+      '/api': {
+        // string shorthand: http://localhost:5173/foo -> http://localhost:4567/foo
+        target: 'http://app:8000',
```

Comment:
```suggestion
        target: 'http://app:8000',
        changeOrigin: true,
        secure: false,
```
Just make sure to have changeOrigin to true. I believe it has to do with the client denying any request, since its going through the vite server then to the api and back.
https://vitejs.dev/config/server-options#server-proxy
Also, have secure to false, it makes things less weird

Reply:
done